nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/model/DocumentModelResolver.java

```diff
@@ -177,16 +177,24 @@ public Object fetch(Object value) throws IllegalStateException {
                 try (CloseableCoreSession session = CoreInstance.openCoreSession(ref.repo)) {
                     try {
                         DocumentModel doc;
+                        DocumentRef documentRef;
+
```

Comment:
`documentRef` -> `docRef`, which is the usual variable name for this in most of our code.